tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +50,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="num_workers", default=NUM_WORKERS, type="int",
```

Comment:
According to the `argparse` documentation the `type` argument should be `int` instead of `"int"`.